tests/bin/build-viewer-reports

```diff
@@ -14,55 +17,33 @@ directory specified on the command line.
 """
 
 from pathlib import Path
-import argparse
 import json
 import logging
 import shutil
 import subprocess
 
 import ninja
```

Comment:
I was missing `ninja`. Installed via `pip install ninja`.

@markrtuttle  @ronakfof  should `ninja` be added as a dependency here?

https://github.com/model-checking/cbmc-viewer/blob/master/setup.cfg#L24-L27

Reply:
ninja is not a requirement for using cbmc-viewer.  there is some regression testing that does, but I wouldn't make it a package dependency.

Reply:
Test dependencies can be specified in `extras_require` option if needed.
https://setuptools.pypa.io/en/latest/userguide/declarative_config.html?highlight=options.extras_require#configuring-setup-using-setup-cfg-files

Reply:
I recommend that we wait until the testing work I've done is part of continuous integration.  Until then, it is basically experimental stuff.